### ISSUE TITLE
Copy link to clipboard

### ISSUE THREAD

**Issue by seejee** [seejee/imgin#6]

**Comment by practicingruby** [comment]
Not sure if this is a good idea, at least by default. If this happens and the person uploading the image doesn't expect it, you might destroy some data in their clipboard. It doesn't seem like that'd be a common problem or anything too tragic, but surprising people is bad. What about an easy to click "copy link to clipboard" button?

**Comment by seejee** [comment]
You're right. We'll start with a button to copy the link to the clipboard. 

Maybe some point down the road we can make it a user preference whether or not to copy the link automatically.

**Comment by jordanbyron** [comment]
I usually don't like copy to clipboard stuff, only because they have to be implemented with flash which I block by default. I totally agree with @sandal that writing automatically is dangerous and probably a bad idea. I'd personally put this towards the back of the TODO list and work on it once you've got the rest of these tickets completed.
